ports/servoshell/desktop/webview.rs

```diff
@@ -393,6 +444,19 @@ where
                     }
                 }
             })
+            .shortcut(CMD_OR_CONTROL, 'W', || {
+                if let Some(id) = self.focused_webview_id {
+                    self.event_queue.push(EmbedderEvent::CloseWebView(id));
+                }
+            })
+            .shortcut(CMD_OR_CONTROL, 'T', || {
+                // data url encoding of: <html><head><title>New Tab</title><body>New Servo Tab!
+                let data_url =
+                    "data:text/html;base64,PGh0bWw+PGhlYWQ+PHRpdGxlPk5ldyBUYWI8L3RpdGxlPjxib2R5Pk5ldyBTZXJ2byBUYWIh";
```

Comment:
Just wondering if it's be better to just keep the default `ServoUrl` that is `servo.org` for the new tab as well, at least for now wdyt?

I like what m2sub suggested but as you said this could be another PR.

Reply:
Ideally it would be really nice to have something like ecosia/duckduckgo as default, but with the current changes, I'm concerned it might make the minibrowser appear less polished or mature. While I recognize that there are still many things we're working on improving, we should try to put the best forward.

Just a thought—I’m happy to hear if others have different opinions